<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RhapsodySelectionView</class>
 <widget class="QDialog" name="RhapsodySelectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Rhapsody Statechart</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select a Rhapsody statechart to be imported:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="importPushButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
